charts/tempo-distributed/README.md

```diff
@@ -604,6 +604,7 @@ The memcached default args are removed and should be provided manually. The sett
 | serviceAccount.create | bool | `true` | Specifies whether a ServiceAccount should be created |
 | serviceAccount.imagePullSecrets | list | `[]` | Image pull secrets for the service account |
 | serviceAccount.name | string | `nil` | The name of the ServiceAccount to use. If not set and create is true, a name is generated using the fullname template |
+| storage.admin.backend | string | `"local"` | The supported storage backends are gcs, s3 and azure, as specified in https://grafana.com/docs/enterprise-traces/latest/config/reference/#admin_client_config |
```

Comment:
`inmem` is also supported. It should be `inmem` instead of `local` as default here.

Reply:
I'd added as the default as the storage was also `local`, but will change this.

Reply:
I see references to inmem in the code, but not sure the support level there.  I think `local` is now `filesystem`, so this has been updated.  Let's come back to the inmem portion.  The `client.go` in mimr only looks to support the ones listed in `NewClient()`.